pages/index.js

```diff
@@ -32,15 +36,13 @@ const Home = ({ doc, posts }) => {
   return <SetupRepo />;
 };
 
-export async function getStaticProps({ preview = null, previewData = {} }) {
-
-  const { ref } = previewData
+export async function getStaticProps({ previewData }) {
 
-  const client = Client()
+  const ref = previewData ? previewData.ref : null
 
-  const doc = await client.getSingle("blog_home", ref ? { ref } : null) || {}
+  const doc = await Client().getSingle("blog_home", ref ? { ref } : null) || {}
 
-  const posts = await client.query(
+  const posts = await Client().query(
     Prismic.Predicates.at("document.type", "post"), {
       orderings: "[my.post.date desc]",
       ...(ref ? { ref } : null)

```

Comment:
Again, here I would set the query options object on the line above to make this line easier to read.